libvast/src/system/spawn_pivoter.cpp

```diff
@@ -0,0 +1,44 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#include "vast/system/spawn_pivoter.hpp"
+
+#include "vast/defaults.hpp"
+#include "vast/detail/unbox_var.hpp"
+#include "vast/error.hpp"
+#include "vast/event_types.hpp"
+#include "vast/logger.hpp"
+#include "vast/query_options.hpp"
+#include "vast/system/node.hpp"
+#include "vast/system/pivoter.hpp"
+#include "vast/system/spawn_arguments.hpp"
+
+#include <caf/actor.hpp>
+#include <caf/expected.hpp>
+#include <caf/send.hpp>
+#include <caf/settings.hpp>
+
+namespace vast::system {
+
+maybe_actor spawn_pivoter(node_actor* self, spawn_arguments& args) {
+  VAST_TRACE(VAST_ARG(args));
+  auto target_name = args.invocation.arguments[0];
+  // Parse given expression.
+  VAST_UNBOX_VAR(
+    expr, normalized_and_valided(std::next(args.invocation.arguments.begin()),
+                                 args.invocation.arguments.end()));
```

Comment:
I find this hard to understand in this local scope without a comment. Can you add a comment about why you call `std::next`?